res/css/views/rooms/_EventTile.scss

```diff
@@ -31,6 +31,10 @@ limitations under the License.
     top: 14px;
     left: 8px;
     cursor: pointer;
+    -webkit-user-select: none;
```

Comment:
We use a build step that adds these vendor prefixes, so your original change with just `user-select` is what we want here.

Reply:
Updated. I've also removed the vendor prefixed user-selects in `mx_EventTile_editButton` (where I initially got the idea that vendor prefixes were needed^^)